MvvmCross/Platforms/Wpf/Presenters/MvxWpfPage.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Windows.Controls;
+using MvvmCross.Platforms.Wpf.Views;
+using MvvmCross.ViewModels;
+
+namespace MvvmCross.Platforms.Wpf.Presenters
```

Comment:
This should be in the `Views` folder. Also it might need some more work, see: https://github.com/MvvmCross/MvvmCross/blob/develop/MvvmCross/Platforms/Wpf/Views/MvxWpfView.cs